### DISCUSSION TITLE
Is this a mistake?

### DISCUSSION THREAD

**Discussion by ryan-reichenberg** [tailwindlabs/tailwindcss#12777] [Help] [answered]
I was looking at the tailwind animations and noticed that you have specified keyframes for the 75% and the 100% positions of the ping animation? I believe the 100% keyframe adds no benefit here?

```
   ping: {
        '75%, 100%': {
          transform: 'scale(2)',
          opacity: '0',
        },
      },
```

If this is intentional can you explain why the CSS animation for the pulse only contains a keyframe for 50%?

```
    pulse: {
      '50%': {
        opacity: '.5',
      },
    },
```

Any answer would be great -- even if it is for my own learning :)

**Comment by adamwathan** [comment, marked as answer]
Hey! Not a mistake, can see the difference in the animation if you remove the `100%` keyframe from the ping animation here:

https://play.tailwindcss.com/nYEYfIBxuQ?file=css

What it does in this case is basically add a pause in the animation from the 75%-100% range, so the ping stays invisible for 25% of the duration of the animation before starting again.

**Reply by ryan-reichenberg** [reply]
Thanks! I didn't realise css animations didn't hold the keyframe unless you had another keyframe specified
